### PR TITLE
First task to development branch

### PR DESCRIPTION
**Pull request by AndriiKolomiiets**

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -77,11 +78,26 @@
...
         </dependency>
+        
         <dependency>
             <groupId>com.adobe.aem</groupId>
             <artifactId>uber-jar</artifactId>
             <classifier>apis</classifier>
         </dependency>
+        <dependency>
+            <groupId>com.adobe.aem</groupId>
+            <artifactId>uber-jar</artifactId>
```
Comment (yktaleb) [inline core/pom.xml]: It is enough to have this dependency in main pom file

File: core/pom.xml [hunk 2]
```diff
@@ -77,11 +78,26 @@
...
+            <version>6.4.0</version>
+            <classifier>apis</classifier>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.geronimo.specs</groupId>
+            <artifactId>geronimo-atinject_1.0_spec</artifactId>
```
Comment (yktaleb) [inline core/pom.xml]: Why did you add this dependency?

File: core/src/test/java/trainingproject/core/models/TestHelloWorldModel.java [hunk 3]
```diff
@@ -51,7 +51,7 @@ public void setup() throws Exception {
     
     @Test
     public void testGetMessage() throws Exception {
-        // some very basic junit tests
+        // some very basic junit aaaaa
```
Comment (yktaleb) [inline core/src/test/java/trainingproject/core/models/TestHelloWorldModel.java]: Pull request should contain only useful changes regarding new feature(task)

File: it.launcher/pom.xml [hunk 4]
```diff
@@ -40,7 +40,7 @@
     </scm-->
 
     <!-- 
-        To keep the instance under test running and run individual tests
+        To keep the instance under test running and run individual aaaaa
```
Comment (yktaleb) [inline it.launcher/pom.xml]: The same

File: ui.apps/src/main/content/META-INF/vault/filter.xml [hunk 5]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
     <filter root="/apps/training"/>
-    <filter root="/apps/sling" />
+    <filter root="/apps/aem-training"/>
+    <filter root="/apps/sling"/>
```
Comment (yktaleb) [inline ui.apps/src/main/content/META-INF/vault/filter.xml]: why did you leave <filter root="/apps/sling"/> and  <filter root="/apps/training"/> ?

File: ui.apps/src/main/content/jcr_root/apps/aem-training/components/topnav/TopNav.java [hunk 6]
```diff
@@ -0,0 +1,31 @@
+package apps.training.components.topnav;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Iterator;
+import com.adobe.cq.sightly.WCMUsePojo;
+import com.day.cq.wcm.api.Page;
+import com.day.cq.wcm.api.PageFilter;
+
+    public class TopNav extends WCMUsePojo {
```
Comment (yktaleb) [inline ui.apps/src/main/content/jcr_root/apps/aem-training/components/topnav/TopNav.java]: ctrl + shift + alt + L in order to reformat file :)

**Review by yktaleb — COMMENTED** [review]
1. Let's remove all packages/files which are not needed under AEM-training-project\ui.apps\src\main\content\jcr_root\apps.
2. Do not see template

File: core/pom.xml (hunk 1 above)
Comment (AndriiKolomiiets) [reply core/pom.xml]: When I remove this dependency all Sling imports goes red. I suppose it is necessary to include it in this pom too.

File: core/pom.xml (hunk 2 above)
Comment (AndriiKolomiiets) [reply core/pom.xml]: I didn't, it was installed automatically when I created project in maven.
I removed it.

File: core/src/test/java/trainingproject/core/models/TestHelloWorldModel.java (hunk 3 above)
Comment (AndriiKolomiiets) [reply core/src/test/java/trainingproject/core/models/TestHelloWorldModel.java]: Got it, removed comments.

File: it.launcher/pom.xml (hunk 4 above)
Comment (AndriiKolomiiets) [reply it.launcher/pom.xml]: Removed.

File: ui.apps/src/main/content/META-INF/vault/filter.xml (hunk 5 above)
Comment (AndriiKolomiiets) [reply ui.apps/src/main/content/META-INF/vault/filter.xml]: Removed all excessive paths, left just /apps/training
